libs/wingsdk/test/target-sim/bucket.test.ts

```diff
@@ -705,3 +705,50 @@ test("tryDelete objects from bucket", async () => {
   expect(existingObject2TryDelete).toEqual(true);
   expect(nonExistentObjectTryDelete).toEqual(false);
 });
+
+// test("Given a bucket when reaching to a non existent key, signed url it should throw an error", async () => {
```

Comment:
Can you please add a comment that says we decided to separate this into a different issue followed by its link- #4143